src/components/group-management/member-management-dialog/index.test.tsx

```diff
@@ -25,99 +28,47 @@ describe(MemberManagementDialog, () => {
     return shallow(<MemberManagementDialog {...allProps} />);
   };
 
-  describe('remove member', () => {
-    it('renders the message with NO room name', function () {
-      const wrapper = subject({ userName: 'Johnny Cash', roomName: '', type: MemberManagementAction.RemoveMember });
-
-      expect(wrapper.find(c('')).text()).toEqual(
-        'Are you sure you want to remove Johnny Cash from the group?Johnny Cash will lose access to the conversation and its history.'
-      );
-    });
-
-    it('renders the message with a room name', function () {
+  describe('confirmition definition (remove member)', () => {
+    it('renders the message', function () {
       const wrapper = subject({
-        userName: 'Johnny Cash',
-        roomName: 'Fun Room',
-        type: MemberManagementAction.RemoveMember,
+        definition: {
+          ...mockConfirmitionDefinition,
+          getMessage: () => (
+            <>
+              Are you sure you want to remove Johnny Cash from Fun Room?
+              <br />
+              Johnny Cash will lose access to the conversation and its history.
+            </>
```

Comment:
since this is a mock anyway you can shorten this up. "test message" proves just as much as a long string